### ISSUE TITLE
Use the script with unpacked sources

### ISSUE THREAD

**Issue by mcepl** [openSUSE/obs-service-go_modules#45]
I have a couple of packages where I follow the master branch of the upstream project, and it is rather inconvenient that I have to `git commit`, `git push`, and `osc services remoterun`, `osc up` to get new `*.obscpio`, so I can run `osc service manualrun`.

See https://build.opensuse.org/package/show/home:mcepl/vis and https://build.opensuse.org/package/show/home:mcepl/aerc.

(and of course I have to have those stupid commit “Re-vendor” all over the package history)

**Comment by mcepl** [comment]
Technically, I would imagine these two additional possible values:

* `compression` would get value `None` or `directory`
* `archive` could just name the subdirectory of the current directory with the unpacked code (default to the name of the package as defined in `Name:` attribute in the SPEC file).

**Comment by jfkw** [comment]
Thanks for raising the server side OBS SCM integration use case. I haven't yet used these modes but would like to support them fully. 

Your [follow up comment](https://github.com/openSUSE/obs-service-go_modules/issues/45#issuecomment-2037002577) clarifies the idea behind this mode. 

Of your two examples:

- `aerc` has `osc meta prj` element `<scmsync>https://git.cepl.eu/git/packaging/vis-pkg</scmsync>`, and `_service` with OBS source service definition: `<service name="go_modules" mode="manual"><param name="compression">zst</param></service>`

- `vis` has `osc meta prj` element `<scmsync>https://git.cepl.eu/git/packaging/vis-pkg</scmsync>` and no `_service` definition.

I presume `vis` does not currently have other configuration to call `obs-service-go_modules`. This is a package we'd like to use `go_modules` with if it does the right thing in scm_sync modes.

For `aerc`, would the plan be to run `go_modules` in one of the server side modes e.g. `buildtime`?

**Comment by mcepl** [comment]
> vis has osc meta prj element <scmsync>https://git.cepl.eu/git/packaging/vis-pkg</scmsync> and no _service definition.

Yes, you are right, sorry, `vis` is probably not relevant to this ticket.

> For `aerc`, would the plan be to run `go_modules` in one of the server side modes e.g. `buildtime`?

I am not sure, I want to be able to build the package both in OBS and locally (`osc build`). If I would have `vendor.tar.*` build only on-the-fly and not cached anywhere, …? Perhaps? Why not?
